Formula/deno.rb

```diff
@@ -32,21 +32,14 @@ def install
     end
 
     # env args for building a release build with our clang, ninja and gn
-    ENV["DENO_NO_BINARY_DOWNLOAD"] = "1"
-    ENV["DENO_GN_PATH"] = buildpath/"gn/out/gn"
-    args = %W[
-      clang_use_chrome_plugins=false
-      mac_deployment_target="#{MacOS.version}"
-      treat_warnings_as_errors=false
-    ]
+    ENV["GN"] = buildpath/"gn/out/gn"
     if DevelopmentTools.clang_build_version < 1100
       # build with llvm and link against system libc++ (no runtime dep)
-      args << "clang_base_path=\"#{Formula["llvm"].prefix}\""
+      ENV["CLANG_BASE_PATH"] = Formula["llvm"].prefix
       ENV.remove "HOMEBREW_LIBRARY_PATHS", Formula["llvm"].opt_lib
     else # build with system clang
-      args << "clang_base_path=\"/usr/\""
+       ENV["CLANG_BASE_PATH"] = "/usr/"
```

Comment:
CI complains about extra space here: 
```
        == /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/deno.rb ==
C: 41:  5: Use 2 (not 3) spaces for indentation.
```

Reply:
Oh, maybe I should actually look at the audit results next time and not just run it and switch to another tab to test out the new test. 😆 But it was late and I was already tired.

Should be fixed now.